packages/serve/src/index.ts

```diff
@@ -1,10 +1,45 @@
-// Test dev-server availability before importing it
+import WebpackCLI from 'webpack-cli';
+import logger from 'webpack-cli/lib/utils/logger';
+import startDevServer from './startDevServer';
+
+let devServerFlags: object[];
 try {
-    // eslint-disable-next-line node/no-extraneous-require
-    require.resolve('webpack-dev-server');
-} catch (error) {
-    throw new Error(`You need to install 'webpack-dev-server' for running 'webpack serve'.\n${error}`);
+    devServerFlags = require('webpack-dev-server/bin/cli-flags').devServer;
+} catch (err) {
+    throw new Error(`You need to install 'webpack-dev-server' for running 'webpack serve'.\n${err}`);
```

Comment:
Put this inside `export default function serve`, because it is break webpack-cli for developers who don't want to use `webpack-dev-server` (and don't install this)

Reply:
Done